.github/workflows/build_linux_pytorch_wheels.yml

```diff
@@ -91,3 +91,18 @@ jobs:
           # These wheels share the same S3 directory as rocm-sdk to simplify --find-links usage
           aws s3 cp ${{ env.PACKAGE_DIST_DIR }}/ s3://therock-nightly-python/${{ inputs.AMDGPU_FAMILIES }}/ \
             --recursive --exclude "*" --include "*.whl"
+
+      - name: Add Python 3.12 to PATH
+        if: ${{ github.repository_owner == 'ROCm' }}
+        run: echo "/opt/python/cp312-cp312/bin" >> $GITHUB_PATH
+
+      - name: (Re-)Generate Python package release index
+        if: ${{ github.repository_owner == 'ROCm' }}
+        run: |
+          pip install boto3
+          python ./build_tools/packaging/python/generate_release_index.py \
+            --bucket=therock-nightly-python \
+            --endpoint=s3.us-east-2.amazonaws.com \
```

Comment:
I think this might not be correct as you're passing `find_links` to the workflow which could point to a dev release which would be in  a different bucket. You would also want to upload those then to the dev bucket and not to the ones for nightlies. This is already a problem when uploading the wheels.

Reply:
I see the role in AWS as `therock-dev-releases`. Can you provide me the link for that?

Tried the link below it did not work. Something I am missing.
https://therock-dev-releases.s3.us-east-2.amazonaws.com/gfx94X-dcgpu/index.html

So overall, I need to use `rocm-sdk` from `dev `not `nightly `

Reply:
Little confused

Yes, currently, I use       
`find_links: https://therock-nightly-python.s3.us-east-2.amazonaws.com/${{ matrix.config.AMDGPU_FAMILIES }}/index.html` to get `rocm-sdk`

And I also push `therock-nightly-python` the Wheels